src/map/plotly/fix-plot.ts

```diff
@@ -0,0 +1,107 @@
+import { PlotlyScatterElement } from './plotly-scatter';
+
+export default function fixPlot(plot: PlotlyScatterElement) {
```

Comment:
Please add a comment explaining what is the issue and why this function is required, ideally linking to the plotly issue